src/Adapter/Image/ImageRetriever.php

```diff
@@ -165,47 +170,56 @@ public function getImage($object, $id_image)
         $urls = [];
         $image_types = ImageType::getImagesTypes($type, true);
         $generateHighDpiImages = (bool) Configuration::get('PS_HIGHT_DPI');
-        $ext = 'jpg';
 
-        // Get path of original uploaded image we will use to get thumbnails
+        // Get path of original uploaded image we will use to get thumbnails (original image is always .jpg)
```

Comment:
I don't understand why we should keep the jpg all the time, the source should use same extension as its format, but I believe you already discussed this and for now it must remain this way I think this should be improved in 9.0 though even if it's a breaking change